scripts/suite.py

```diff
@@ -35,6 +35,11 @@
                               If both -S and -c are supplied, -S wins, and
                               there is no validation that sha1 is contained
                               in branch
+  -n, --newest                Search for the newest revision built on all
+                              required distro/versions, starting from
+                              either --ceph or --sha1.
+  --newest-limit <backtrack>  backtrack is the number of commits to try
+                              in --newest mode  [default: 10]
```

Comment:
Right, and I couldn't figure out how to make docopt have an optional arg with a default.

Sent from Nine
________________________________
From: Zack Cerza <notifications@github.com>
Sent: Jun 23, 2016 9:47 AM
To: ceph/teuthology
Cc: Dan Mick; Author
Subject: Re: [ceph/teuthology] Add teuthology-suite --newest (#883)

In scripts/suite.py:

> @@ -35,6 +35,11 @@
>                                If both -S and -c are supplied, -S wins, and
>                                there is no validation that sha1 is contained
>                                in branch
> +  -n, --newest                Search for the newest revision built on all
> +                              required distro/versions, starting from
> +                              either --ceph or --sha1.
> +  --newest-limit <backtrack>  backtrack is the number of commits to try
> +                              in --newest mode  [default: 10]

Are these separate because you want to be able to pass -n and not e.g. -n 10 all the time?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.